timemory/roofline/roofline.py

```diff
@@ -521,8 +852,10 @@ def __init__(self, peak_flops, hotspots):
         self.xmax = 1000
         self.ymin = 1
         self.ymax = 10**y_digits
-        self.xlabel = ('Arithmetic Intensity [FLOPs/Byte]')
-        self.ylabel = ('Performance [GFLOPs/sec]')
+        #self.xlabel = ('Arithmetic Intensity [FLOPs/Byte]')
```

Comment:
There needs to be logic here. We want to adapt the label to what is being plotted